user_manual/files/access_webgui.rst

```diff
@@ -97,6 +96,17 @@ Overflow Menu
   
 .. figure:: ../images/files_page-4.png
    :alt: Details screen.  
+   
+The **Settings** gear icon at the lower left allows you to show or hide hidden 
+files in your ownCloud Web interface. These are also called dotfiles, because 
+they are prefixed with a dot, e.g. ``.mailfile``. The dot tells your operating 
+system to hide these files in your file browsers, unless you choose to display 
+them. Usually these are configuration files, so having the option to hide them 
+reduces clutter. When you enable or disable showing hidden files, refresh your 
+Web browser to see the change.
```

Comment:
Ubuntu 14.04 with Firefox. The first few times I tried it I had to refresh to see the changes. Now I don't. This is why I love computers.